### ISSUE TITLE
Walking causes screen to flip and flicker

### ISSUE THREAD

**Issue by Puchor** [NillerMedDild/Enigmatica2Expert#455]
## Thank you for helping us improve. #####

_Please follow the issue template, and do not delete any part of it._
_Failure to comply will result in the issue being closed._

**Issue Description**
Walking around cause screen to flip around like crazy and flash random shade of colors. If this helps the color has to do with whatever color may be on my screen at the moment. The collor tends to be a light blue like the sky. It's only when I walk around

**Modpack version (Do not use "latest")**
Enigmatica 2: Expert 1.46

**Logs**

If applicable, please upload your crash report, or latest.log to [hastebin](https://hastebin.com/).

**Screenshots**

**Additional context**

**Comment by drakkensong21** [comment]
Did you change the algorithms in the pause menu

**Comment by Puchor** [comment]
Yes I did earlier today, I got it to work. The pack was having a lot of performance issues and I have a really, really good PC. After allocating around 7-8 GB of RAM it still wasn’t working. I just tinkered with the video settings until my game was running stable but then I got that flickering problem. So after I changed my algorithm to something along the lines of “Riven ‘half’ algorithm” it started working fine. Thank you for replying by the way, loving the pack so far!
